packages/nodes-base/nodes/Webhook/Webhook.node.ts

```diff
@@ -126,6 +126,10 @@ export class Webhook implements INodeType {
 						name: 'None',
 						value: 'none',
 					},
+					{
+						name: 'Webhook Auth',
+						value: 'webhookAuth',
```

Comment:
@daylay92  make it more specific:
name: NocoDB webhook auth
value: nocodbWebhookAuth